README.md

```diff
@@ -24,7 +24,7 @@ julia> add_artifact!(
                      "Artifacts.toml",
                      "JuliaMono",
                      "https://github.com/cormullion/juliamono/releases/download/v0.030/JuliaMono.tar.gz",
-                     force=true,
+                     force=true
```

Comment:
Why erroneous?  Trailing commas are useful not to break git blame, you can add new options below without touching this line